### ISSUE TITLE
Multi-line struct assignment

### ISSUE THREAD

**Issue by rgreenjr** [lexmag/elixir-style-guide#45]
I'd appreciate verification on the preferred way to wrap multi-line struct assignments. The three possible scenarios are depicted below.

Thanks in advance!

1. Left hand side fits, but right doesn't:

```elixir
datetime =
  %NaiveDateTime{calendar: Calendar.ISO, day: 1, month: 12, year: 2017}
```

2. Right hand side fits, but left doesn't:

```elixir
%NaiveDateTime{calendar: calendar, day: day, month: month, year: year} =
  datetime_struct
```

3. Neither right nor left hand side fits:

```elixir
%NaiveDateTime{
  calendar: calendar,
  day: day,
  hour: hour,
  microsecond: microsecond,
  minute: minute,
  month: month,
  second: second,
  year: year} = 
  %NaiveDateTime{
    calendar: Calendar.ISO,
    day: 1,
    hour: 23,
    microsecond: {0, 0},
    minute: 59,
    month: 1,
    second: 59,
    year: 2017
  }
```

**Comment by lpil** [comment]
Any preference here? I'm thinking this:

Left hand side fits, but right doesn't:

```elixir
datetime = %NaiveDateTime{
  calendar: Calendar.ISO, 
  day: 1, 
  month: 12, 
  year: 2017,
}
```
Right hand side fits, but left doesn't:

```elixir
%NaiveDateTime{calendar: calendar, day: day, month: month, year: year} =
  datetime_struct
```

Neither right nor left hand side fits:

```elixir
%NaiveDateTime{
  calendar: calendar,
  day: day,
  hour: hour,
  microsecond: microsecond,
  minute: minute,
  month: month,
  second: second,
  year: year,
} = %NaiveDateTime{
  calendar: Calendar.ISO,
  day: 1,
  hour: 23,
  microsecond: {0, 0},
  minute: 59,
  month: 1,
  second: 59,
  year: 2017,
}
```

**Comment by rgreenjr** [comment]
Closing now that the code formatter has been released in Elixir v1.6.
